server/server.js

```diff
@@ -21,16 +22,25 @@ app.post('/sign-up', upload.single('photo'), (req, res) => {
 });
 
 // Log in
-app.post('/log-in', (req, res) => {
+app.post('/log-in', upload.single('photo'), (req, res) => {
```

Comment:
Use contants for urls of all endpoints and hardcoded URL on the server and client sides.